docs/SMCWG-charter.md

```diff
@@ -0,0 +1,328 @@
+     trust.
+
+   Applicants that qualify as Certificate Issuers or Root Certificate Issuers
+   must supply the following additional information:
+
+   * URL of the current qualifying audit report.
+   * Links or references to issued end-entity certificates that demonstrate
+     them being treated as valid by a Certificate Consumer Member.
+
+   Such Applicant SHALL become a Member once the SMCWG has determined by
+   consensus among the Members during a SMCWG Meeting or Teleconference that
+   the Applicant meets all of the requirements above or, upon the request of
+   any Member of the SMCWG, by a Ballot among Members of the SMCWG. Acceptance
+   by consensus shall be determined or a Ballot of the Members shall be held
+   as soon as the Applicant indicates that it has presented all information
+   required above and has responded to all follow-up questions from the SMCWG
+   and the Member has complied with the requirements of Bylaw 5.5.
+
+   Certificate Issuer applicants that are not actively issuing S/MIME
+   certificates but otherwise meet these membership criteria MAY request to the
+   SMCWG that they be granted an invitation for Associate Member status in
+   accordance with Bylaw 3.1, subject to conditions designated by the SMCWG.
+
+2. Existing CAB Forum Members seeking to participate in the SMCWG, in accordance
+   to Bylaw 5.3.1(c), MUST formally declare their intent to participate in
+   writing and provide the SMCWG Chair with this declaration and evidence that
+   they meet the criteria set forth above. Such Applicants SHALL become Members
+   of the SMCWG as determined by consensus during a SMCWG Meeting or
+   Teleconference, or upon the request of any Member of the SMCWG, by a Ballot
+   among Members of the SMCWG.
+
+   In order to determine the composition of the initial set of SMCWG Members,
+   at least twenty-four (24) hours prior to the initial meeting of the SMCWG,
+   the SMCWG Chair SHALL publish a list of Members seeking to participate who
+   they determine meet the criteria set forth above. As the first order of
+   business at the first meeting of the SMCWG, those organizations on the
+   Chair’s list of proposed, qualifying Members SHALL vote to determine the
+   initial set of SMCWG Members.
+
+##### 4.2.3. Suspension and Cessation of Working Group Membership
+
+Members may resign from the SMCWG at any time. Resignation, suspension, or
+other termination of membership in the SMCWG does not prevent a Member from
+potentially having continuing obligations, under the Forum’s IPR Policy or any
+other document.
+
+A Certificate Consumer Member’s membership may be suspended if any of the
+following become true:
+
+1. it stops providing support for its membership-qualifying software
+   product;
+1. its membership-qualifying software product ceases to consume
+   S/MIME certificates;
+1. it announces an end of life for its membership-qualifying software product
+   and said end of life occurs.
```

Comment:
This third bullet point does feel duplicative of the first, at least in how it would likely be realized in practice where no shenanigans take place. I don't think it's necessary to keep either way and worried even while writing it about introducing the concept of "end of life".